plugin/rocks.lua

```diff
@@ -18,6 +18,7 @@ end
 
 local log = require("rocks.log")
 log.info(vim.uv.os_uname())
+log.info("Using luarocks config " .. (vim.g.rocks_nvim.luarocks_config or "unset"))
```

Comment:
issue: we should get the luarocks config from the `config` module (see L27)

Reply:
ha yes ty. I noticed that after the initial implementation and forgot to correct it. The CI error looks unrelated